azavea/pfb-network-connectivity
# Feature/tnation/terraform 0.9

## Overview
Updates Terraform to 0.9.x, which improves remote state management by adding a `backend` configuration object. Remote state is now managed via  `terraform init` instead `terraform remote config`.

Fixes #172

## Testing Instructions

 * Delete `deployment/terraform/.terraform`
 * Run `scripts/infra.sh plan`, ensure no changes are necessary.
```bash
vagrant@vagrant-ubuntu-trusty-64:/vagrant$ export PFB_SETTINGS_BUCKET=staging-pfb-config-us-east-1
vagrant@vagrant-ubuntu-trusty-64:/vagrant$ ./scripts/infra.sh plan
No changes. Infrastructure is up-to-date.

This means that Terraform did not detect any differences between your
configuration and real physical resources that exist. As a result, Terraform
doesn't need to do anything.

```
